app/controllers/spree/admin/adjustments_controller.rb

```diff
@@ -28,6 +29,10 @@ def set_order_id
         @adjustment.order_id = parent.id
       end
 
+      def skip_changing_canceled_orders
+        redirect_to admin_order_adjustments_path(@order) if @order.canceled?
```

Comment:
It feels like a flash message would be helpful here to give some user feedback.

Reply:
I added a commit for this.